qa/workunits/rbd/cli_migration.sh

```diff
@@ -69,6 +76,21 @@ remove_images() {
     done
 }
 
+remove_namespace() {
```

Comment:
It occurs to me that we could get rid of this function by defining
```
IMAGES="${IMAGE1} ${IMAGE2} ${IMAGE3} ${NAMESPACE1}/${IMAGE1} ${NAMESPACE2}/${IMAGE2}"
```
and changing `remove_namespaces()` definition to:
```
remove_namespaces() {
    for namespace in ${NAMESPACES}
    do
        rbd namespace remove rbd/${namespace} || true
    done
}
```

Reply:
Thanks for this logic, now made the changes accordingly.